exchanges/lbank/lbank_types.go

```diff
@@ -183,10 +184,15 @@ type ExchangeRateResponse struct {
 
 // WithdrawConfigResponse stores info about withdrawal configurations
 type WithdrawConfigResponse struct {
-	AssetCode   string `json:"assetCode"`
-	Minimum     string `json:"min"`
-	CanWithDraw bool   `json:"canWithDraw"`
-	Fee         string `json:"fee"`
+	AmountScale         int64         `json:"amountScale,string"`
+	Chain               string        `json:"chain"`
+	AssetCode           currency.Code `json:"assetCode"`
+	Minimum             float64       `json:"min,string"`
+	TransferAmountScale int64         `json:"transferAmountScale,string"`
+	CanWithDraw         bool          `json:"canWithDraw"`
```

Comment:
🥃 Probably not worth breaking user compat, but
```suggestion
	CanWithdraw         bool          `json:"canWithDraw"`
```